project_euler/problem_16/sol2.py

```diff
@@ -25,4 +27,4 @@ def solution(power):
 
 
 if __name__ == "__main__":
-    print(solution(int(str(input()).strip())))
+    print(f"Sum of the digits is {solution(1000)}")
```

Comment:
Same as above

Reply:
my bad, thought that without input() it will be better. Like in some other solutions in problems.